Formula/telnet.rb

```diff
@@ -0,0 +1,46 @@
+class Telnet < Formula
+  desc "Apple telnet built from macOS 10.12.4 sources"
```

Comment:
Let's avoid hard-coding the version number in the description but you can mention the sources are from Sierra.